drivers/ethernet/Kconfig.native_posix

```diff
@@ -0,0 +1,83 @@
+# Kconfig - Native posix ethernet driver configuration options
+
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig ETH_NATIVE_POSIX
+	bool
+	prompt "Native Posix Ethernet driver"
+	depends on ARCH_POSIX
+	select NET_L2_ETHERNET
+	default n
+	help
+	  Enable native posix ethernet driver. Note, this driver is run inside
+	  a process in your host device.
+
+if ETH_NATIVE_POSIX
+config ETH_NATIVE_POSIX_SETUP_SCRIPT
+	string "Setup script"
+	default "${ZEPHYR_BASE}/samples/net/eth_native_posix/net_setup_host"
+	help
+	  This option sets the name of the script that is run when interface is
+	  created. The script should setup IP addresses etc for the host TAP
+	  network interface.
+	  The script will be called with "<host interface name>" as a parameter.
+
+config ETH_NATIVE_POSIX_DRV_NAME
+	string "Ethernet driver name"
+	default "zeth"
+	help
+	  This option sets the driver name and name of the interface in your
+	  host system.
+
+config ETH_NATIVE_POSIX_DEV_NAME
+	string "Host ethernet TUN/TAP device name"
+	default "/dev/net/tun"
```

Comment:
Just a comment: It would seem to me that a few of these options would benefit from a command line switch to set them (or override their default). When #6138 is merged it should be easy to add them.

Reply:
Sure, makes sense. I would like to keep also possibility to configure this via zephyr config file too.